apps/api/src/app/integrations/e2e/get-active-integration.e2e.ts

```diff
@@ -0,0 +1,87 @@
+import { UserSession } from '@novu/testing';
+import { expect } from 'chai';
+import { ChannelTypeEnum, EmailProviderIdEnum } from '@novu/shared';
+import { IntegrationRepository } from '@novu/dal';
+
+describe('Get Active Integrations - /integrations/active (GET)', function () {
+  let session: UserSession;
+  const integrationRepository = new IntegrationRepository();
+
+  beforeEach(async () => {
+    session = new UserSession();
+    await session.initialize();
+  });
+
+  it('should get active integrations', async function () {
+    const activeIntegrations = (await session.testAgent.get(`/v1/integrations/active`)).body.data;
+
+    const { inAppIntegration, emailIntegration, smsIntegration, chatIntegration, pushIntegration } =
+      splitByChannels(activeIntegrations);
+
+    expect(inAppIntegration.length).to.equal(1);
+    expect(emailIntegration.length).to.equal(1);
+    expect(smsIntegration.length).to.equal(1);
+    expect(pushIntegration.length).to.equal(1);
+    expect(chatIntegration.length).to.equal(2);
+
+    expect(inAppIntegration[0].selected).to.equal(true);
+    expect(emailIntegration[0].selected).to.equal(true);
+    expect(smsIntegration[0].selected).to.equal(true);
+    expect(pushIntegration[0].selected).to.equal(true);
+
+    const oneSelected = chatIntegration.some((integration) => integration.selected === true);
+    const oneNotSelected = chatIntegration.some((integration) => integration.selected === false);
+
+    expect(oneSelected).to.equal(true);
+    expect(oneNotSelected).to.equal(true);
+  });
+
+  it('should have thrown an exception if no active integration are exist', async function () {
+    const activeIntegrations = (await session.testAgent.get(`/v1/integrations/active`)).body.data;
+    await deleteAll(activeIntegrations, integrationRepository, session);
+    const response = await session.testAgent.get(`/v1/integrations/active`);
+
+    expect(response.body.statusCode).to.equal(404);
```

Comment:
I think we shouldn't send a 404 as we are trying to retrieve a list of resources that match certain conditions instead of checking a certain resource.
I don't recall in our API to have any get endpoint that for lack of results in a group of resources `/resources` to return a 404 and I haven't experienced an API designed like that so I am more keen in return a 200 with an empty array as we already do.
I do understand the intention but in this case I think 404 is not the best response to send even if at the end is _we haven't found the resources we were looking for_.

Reply:
good catch! actually i was sure i saw it somewhere in the app we do, but don't find any place we throw exceptions, will update to an empty array.